openedx/core/djangoapps/notifications/models.py

```diff
@@ -5,6 +5,19 @@
 from django.db import models
 from model_utils.models import TimeStampedModel
 
+from opaque_keys.edx.django.models import CourseKeyField
+
+
+NOTIFICATION_PREFERENCE_CONFIG_DEFAULT = {
+    "discussion": {
+        "new_post": {
+            "web": False,
+            "push": False,
+            "email": False,
+        },
+    },
+}
+
```

Comment:
We would also need a CONFIG_VERSION const so that it is updated every time the configuration is updated